qbft/spectest/tests/controller/latemsg/late_commit.go

```diff
@@ -10,17 +10,20 @@ import (
 
 // LateCommit tests process late commit msg for an instance which just decided
 func LateCommit() *tests.ControllerSpecTest {
-	identifier := types.NewMsgID(testingutils.TestingValidatorPubKey[:], types.BNRoleAttester)
 	ks := testingutils.Testing4SharesSet()
 
-	msgs := testingutils.DecidingMsgsForHeight([]byte{1, 2, 3, 4}, identifier[:], qbft.FirstHeight, ks)
-	msgs = append(msgs, testingutils.SignQBFTMsg(ks.Shares[4], 4, &qbft.Message{
-		MsgType:    qbft.CommitMsgType,
-		Height:     qbft.FirstHeight,
-		Round:      qbft.FirstRound,
-		Identifier: identifier[:],
-		Data:       testingutils.CommitDataBytes([]byte{1, 2, 3, 4}),
-	}))
+	msgs := []*qbft.SignedMessage{
+		testingutils.TestingProposalMessage(ks.Shares[1], 1),
+
+		testingutils.TestingPrepareMessage(ks.Shares[1], 1),
+		testingutils.TestingPrepareMessage(ks.Shares[2], 2),
+		testingutils.TestingPrepareMessage(ks.Shares[3], 3),
+
+		testingutils.TestingCommitMessage(ks.Shares[1], 1),
+		testingutils.TestingCommitMessage(ks.Shares[2], 2),
+		testingutils.TestingCommitMessage(ks.Shares[3], 3),
+	}
+	msgs = append(msgs, testingutils.TestingCommitMessage(ks.Shares[4], types.OperatorID(4)))
```

Comment:
@aaron-blox It's done according to the change on the screenshot in https://github.com/alonmuroch/ssv-spec. I haven't checked if there's any difference in the behavior, and it was quicker for me to do the same change as well. I'll check this when I'll be fixing the `LateCommit` test.
<img width="1359" alt="image" src="https://user-images.githubusercontent.com/24493220/219032344-95ef2eaf-a217-4f25-8bcd-dd2f24682b00.png">

Reply:
@aaron-blox I just checked why it did that: The signature of `DecidingMsgsForHeight` was changed and needs passing `*types.ConsensusData` instead of `[]byte` as the first argument, then `*types.ConsensusData` is hashed inside `DecidingMsgsForHeight` and the hash is used as a root. But we use `TestingQBFTRootData` in tests as a root, so the `DecidingMsgsForHeight` function needs to be adjusted to be able to use it here.